src/styles/index.scss

```diff
@@ -1,3 +1,258 @@
+@import './variables';
+
+/* #region page */
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  display: flex;
+  flex-direction: column;
 }
+
+/* #endregion */
+
+/* #region header */
+header {
+  height: auto;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: relative;
+  padding: 0 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  display: inline-block;
+  position: relative;
+  padding: 10px 0;
+}
+
+/* #endregion */
+
+/* #region nav */
+.nav_list {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+li {
+  margin-left: 20px;
+}
+
+.nav_link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  color: #060b35;
+  font-weight: 500;
+  height: 60px;
+  text-align: center;
+  margin-right: 50px;
+  text-transform: uppercase;
+  transition: color 0.3s ease;
+
+  &:hover {
+    color: #00acdc;
+  }
+
+  &.is-active {
+    color: #00acdc;
+
+    &.is-active::after {
+      content: '';
+      bottom: 0;
+      left: 0;
+      right: 0;
+      position: absolute;
+      background-color: #00acdc;
+      height: 4px;
+      border-radius: 8px;
+    }
+  }
+
+  &:first-child {
+    margin-left: 0;
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+}
+
+.nav_list .nav_link:nth-child(4):hover {
+  color: #00acdc;
+}
+
+/* #endregion */
+
+/* #region stars */
+.stars {
+  display: flex;
+  width: 100%;
+  max-width: 250px;
+  justify-content: flex-start;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-size: cover;
+    margin-right: 4px;
+  }
+
+  &--4 .stars__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+/* #endregion */
+
+/* #region main */
+main {
+  margin-top: 50px;
+}
+
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  gap: 46px;
+  margin-bottom: 48px;
+  justify-items: center;
+}
+
+.card {
+  width: 200px;
+  height: auto;
+  padding: 16px;
+  border-radius: 5px;
+  color: $main-text-color;
+  box-sizing: border-box;
+  transition:
+    transform 0.3s ease,
+    box-shadow 0.3s ease;
+  position: relative;
+
+  &:hover {
+    transform: scale(1.2);
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  }
+
+  &__title:hover {
+    color: #34568b;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__describe {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    margin-bottom: 4px;
+    margin-top: 40px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary-color;
```

Comment:
Ensure that the variable `$secondary-color` is defined in the imported `variables` file. If not, this will cause an error during compilation.